src/main/java/com/yahoo/bullet/pubsub/PubSubMessage.java

```diff
@@ -0,0 +1,87 @@
+package com.yahoo.bullet.pubsub;
+
+import java.io.Serializable;
+import com.yahoo.bullet.pubsub.Metadata.Signal;
+import lombok.Getter;
+
+@Getter
+public class PubSubMessage implements Serializable {
+    private String id;
+    private String content;
+    private Metadata metadata;
+    private long sequenceNumber;
+
+    /**
+     * Constructor for a message having only content.
+     *
+     * @param id is the query ID associated with the message.
```

Comment:
We shouldn't mention that this is the queryID everywhere. I think we should just a class javadoc that mentions how PubSubMessage can be used with id being a query id and sequence being multi-part atomic messages per query.